.github/workflows/publish.yaml

```diff
@@ -0,0 +1,40 @@
+name: Publish
+
+on: workflow_dispatch
+
+jobs:
+  cargo-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.version_tag }}
+
+      - name: Fetch tags
+        run: git fetch --prune --unshallow --tags
+
+      - name: Install linux build requirements
+        run: sudo apt install --yes --no-install-recommends libasound2-dev pkg-config
+
+      - name: Publish and tag
+        run: |
+          echo "Current git commit is $(git rev-list -n 1 HEAD)."
+          VERSION="$(yq '.package.version' Cargo.toml)"
```

Comment:
`yq` is installed by defatult in Ubuntu. Unfortunately it cannot write TOML yet. If it could we could also automatically update the cargo version to something like`major.minor.(patch+1)-dev`.

Reply:
lots cleaner then the grep | cut etc hack we had here, nice!